frontend/src/concepts/pipelines/apiHooks/mlmd/__tests__/usePipelineRunMlmdContext.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { testHook, standardUseFetchState } from '~/__tests__/unit/testUtils/hooks';
+import { MlmdContextTypes } from '~/concepts/pipelines/apiHooks/mlmd/types';
+import { useMlmdContext } from '~/concepts/pipelines/apiHooks/mlmd/useMlmdContext';
+import { usePipelineRunMlmdContext } from '~/concepts/pipelines/apiHooks/mlmd/usePipelineRunMlmdContext';
+import { Context } from '~/third_party/mlmd';
+
+// Mock the useMlmdContext hook
+jest.mock('~/concepts/pipelines/apiHooks/mlmd/useMlmdContext', () => ({
+  useMlmdContext: jest.fn(),
+}));
+
+describe('usePipelineRunMlmdContext', () => {
```

Comment:
Considering `usePipelineRunMlmdContext` is just a wrapper on top of `useMlmdContext` with the type set to `MlmdContextTypes.RUN`, these test cases aren't very useful. In fact they stood out to me as being odd because they don't require waiting for an update since you've mocked the one call made to `useMlmdContext`. Essentially all the tests do is test your mock.

Reply:
i will remove it